templates/include/glean/url.1.schema.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "string"
```

Comment:
Should we validate against a regex `pattern` here to enforce the same rules that are enforced at the SDK level per the [URL validation section of the proposal doc](https://docs.google.com/document/d/1kE4wV5qq-N4h5YZSft98-f3iVwt8RKkVWua8LblufC8/edit#heading=h.8tzmxuxhqe6m)?

Reply:
Hm, that sounds reasonable. I am actually changing validation logic a bit on the SDK side, once that is finished I'll add a new regex here.